contracts/staking/tests/Unpack.t.sol

```diff
@@ -0,0 +1,88 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+// SPDX-License-Identifier: UNLICENSED
+// slither-disable-next-line solc-version
+pragma solidity ^0.8.0; // solhint-disable-line compiler-version
```

Comment:
any particular reason not to go with `0.8.25` like the other contracts?

Reply:
can we stick with `0.8.25` here? Mostly want the repo to stick with the same version to manage.

Reply:
I've added a flag to allow for an explicit version, and set it to `0.8.25` in the `//go:generate` directive.